README.md

```diff
@@ -18,51 +16,50 @@ A star ⭐️ is welcome.
 [![GitPOAP Badge](https://public-api.gitpoap.io/v1/repo/dbpunk-labs/db3/badge)](https://www.gitpoap.io/gh/dbpunk-labs/db3)
 [![Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.gg/sz3bmZx2uh)
 
-
 **English | [中文](./README_zh_cn.md)**
 
-# What's DB3 Network
+## What's DB3 Network?
 
-DB3 Network is an open-source decentralized firebase firestore alternative to build dapps quickly with minimal engineering effort.
+DB3 Network is an open-source, decentralized firebase firestore alternative to build dApps quickly and less challenging.
 
-![position_web3](./docs/images/position_web3.jpg)
-
-# Why DB3 Network
+<p align="left">
+ <img width="600px" src="./docs/images/position_web3.svg" align="center"/>
 
+## Why DB3 Network
 ![why db3](./docs/images/why_db3.jpg)
 
-when you want to build a dapp that has custom data to store and query, you have two choices at currently
-
-1st. use a centralized data architecture .e.g firebase or MongoDB to store the custom data. this is friendly for developers but will break the law of decentralization
+Currently, there are two types of Data architecture for dApp(decentralized application): centralized vs. decentralized.
+  
+**Centralized**: use [Firebase](https://firebase.google.com)<img height="20" width="20" src="https://cdn.jsdelivr.net/npm/simple-icons@v8/icons/firebase.svg" />
+ or [MongoDB](https://github.com/mongodb/mongo)<img height="20" width="20" src="https://cdn.jsdelivr.net/npm/simple-icons@v8/icons/mongodb.svg" />to store the data), both of which are developer-friendly. However, dApps would be regarded as less secure based on a central database.
```

Comment:
`less secure` is not clear to deliver the disadvantage of centralized data architecture

Reply:
"The drawbacks of centralized architecture will gradually emerge."